src/hmem_cuda.c

```diff
@@ -760,6 +763,19 @@ bool cuda_is_gdrcopy_enabled(void)
 	return hmem_cuda_use_gdrcopy;
 }
 
+#if CUDA_VERSION >= 11030
+int cuda_flush_rdma_operations(void)
+{
+	return cuda_ops.cudaDeviceFlushGPUDirectRDMAWrites(cudaFlushGPUDirectRDMAWritesTargetCurrentDevice,
```

Comment:
CUDAIsAsBadAsWindowsForExcessivelyLongNamesThatAreAnEntireManPage.  Please move the parameter to the next line to keep the ending character closer to 80 characters.  Long lines are hard to read and frequently requires that the reader scroll left/right, which is a pain.  This comment applies to the function declaration above in cuda_ops as well.

Reply:
done